test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -42,5 +44,21 @@ var server = net.createServer(function(s) {
 
   // Send 2 handles to make `process.disconnect()` wait
   send();
-  send();
+  var s = send(function(err) {
+    // Ignore errors when sending the second handle because the worker
+    // may already have exited.
+    if (err) {
```

Comment:
Perhaps it would be better to at least check that it's a possibly expected error?